templates/common/infra/bicep/core/host/container-app-upsert.bicep

```diff
@@ -67,7 +70,7 @@ param serviceBinds array = []
 @description('The target port for the container')
 param targetPort int = 80
 
-resource existingApp 'Microsoft.App/containerApps@2023-05-02-preview' existing = if (exists) {
+resource existingApp 'Microsoft.App/containerApps@2023-04-01-preview' existing = if (exists) {
   name: name
 }
```

Comment:
We are we choosing to use a lower preview version here?
Is the version supported by gov clouds?  

We should discuss with PM whether or not all our templates are expected to work across all clouds or not.